rfc/manualjudgment.md

```diff
@@ -0,0 +1,68 @@
+# Select roles that can execute manual judgement stage
+
+| | |
+|-|-|
+| **Status**     | _**Proposed**, Accepted, Implemented, Obsolete_ |
+| **RFC #**      | https://github.com/spinnaker/deck/pull/7636
+| **Author(s)**  | Sanjeev Thatiparthi (@sanopsmx) (https://github.com/sanopsmx)
+| **SIG / WG**   | sig-ux
+
+## Overview
+
+Display all the roles the user have in the manual judgment stage.
+
+### Goals and Non-Goals
+
+Display authorized groups dropdown list only when the user selects the manual judgment stage.
+Display all the groups the user has selected while creating application(permissions) in the authorized groups dropdown.
+When the pipeline is executed, disable the manual judgment 'continue/cancel' based on the user role selected in the authorized groups.
+
```

Comment:
Which of these are goals and which are non-goals? Could you make that more clear?

Some higher level questions:
* Will a user be able to opt-out of (or opt-into) this feature?
* If I don't have Authz enabled, will it continue to function as it does today?
* How will we account for the case where the user building a pipeline is _different_ from the user approving the Manual Judgement? Example - a DevOps team member is responsible for building a pipeline but an Eng Manager is the one responsible for approving the stage. Assume the DevOps user doesn't have the same role set at the Eng Manager. How does the DevOps user actually configure the pipeline as needed?
* Does this feature handle cases where multiple approvals are needed?